guides/compensation/expenses.md

```diff
@@ -31,4 +35,20 @@ You need to send any VAT receipts for items purchased with the company credit ca
 
 If you have purchased an item with your own money that is listed above, you can submit this expense and you will be compensated via bank transfer when expenses are processed.
 
-You need to submit these expenses in Xero, our accounting system. There is a [guide on submitting expenses](https://help.xero.com/uk/Payments_AddClaimReceipt) on Xero's help site. You will need to ensure you upload a picture of a VAT receipt with your claim.
+You need to submit these expenses via the Xero Expenses app. You can download this to your phone from App Store/Google Play.
+
+To get reimbursed:
+
+ * You need to submit a receipt that shows the VAT or VAT number. Please do not submit bank statements etc.
+ * You need to include a suitably detailed description: Instead of "Travel" say "Train to Glasgow for Housing Scotland Review"
+ * You need to select the name of the delivery from the "Client" list. Do not select from "Customer". (This is confusing - be vigilant.)
+ * You need to select a "Business Unit". For delivery related expenses for travel/food etc the correct Business Unit is "Delivery/Operations". For food/drink/social expenses inside Made Tech (e.g. team drinks), the correct unit is "People"
+
+All of the above criteria need to be included in your expense to be accepted by Finance.
+
+You do not need to use this method for expensing subscriptions to services required as part of a delivery, such as Trello/Sentry etc. For these delivery related expenses, please request card details via the [Third Party Expense Form](../../guides/3rd_party_services.md)
+
+## Does Made Tech ever cover the cost of customer-related expenses?
+That depends. In the case of "customer entertainment" (e.g. buying drinks for customers), yes. In all other cases, usually only as a last resort and it requires Director approval.
```

Comment:
Is this clearer? It might not be

That depends. In the case of customer entertainment (e.g. buying drinks for customers), yes. In all other cases, such as signing up for a third party service to support a customer delivery, only as a only as a last resort and only with Director approval.